src/Kunstmaan/MediaBundle/Controller/MediaController.php

```diff
@@ -159,6 +159,11 @@ public function bulkUploadSubmitAction(Request $request, $folderId)
             @\mkdir($targetDir);
         }
 
+        $submittedToken = $request->headers->get('x-upload-token');
```

Comment:
Skipping the check if the header is not present would lead to it being completely useless. The idea is to avoid people being able to send a link to people logged in as admin where they could unknowingly upload media. If people can send the link without the header the security check would not work.